packages/ui-next/src/assets/index.ts

```diff
@@ -0,0 +1,2 @@
+export const PLACEHOLDER =
```

Comment:
Not necessary for this PR, but we should probably move towards including icons as a static asset, so they don't bloat the bundle

Reply:
For some reason I was not able to manage svg in the ui-next package, either not supported or not exported, I tried hard and for so long but there is probably a specific config somewhere avoiding it that I didnt found, I just ended it with the b64 version of the file to unlock the situation